tests/unit/components/EmailTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace prime\tests\components;
+
+use Yii;
+use yii\symfonymailer\Message;
+
+final class EmailTest extends \Codeception\Test\Unit
```

Comment:
Missing `@covers` annotation. What code is this unit test supposed to cover?

There is no `prime\components\Email` class.

It seems like this is mostly testing external code without verifying anything actually works.